app/assets/stylesheets/components/auth-form.scss

```diff
@@ -32,43 +33,43 @@
 
   .lbl {
     font-size: 14px;
-    margin: 0px;
-    line-height: 18px;
     font-weight: bold;
+    line-height: 18px;
+    margin: 0px;
   }
 
   .field {
-    border: 1px solid #dae1e3;
-    font-size: 18px;
-    font-weight: 100;
-    padding: 10px 10px 10px 35px;
+    border: 1px solid $field-border;
     border-radius: 4px;
-    color: #4b5b62;
+    color: $input-color;
     display: block;
+    font-family: inherit;
+    font-size: 18px;
+    font-weight: 100;
     height: 44px;
+    padding: 10px 10px 10px 35px;
     width: 100%;
-    font-family: inherit;
   }
 }
 
 .button {
   border-radius: 5px;
+  box-shadow: 0 1px 0 $box-shadow;
```

Comment:
Properties should be ordered border-radius, -webkit-box-shadow, box-shadow, color, fill, font-size, font-weight, height, line-height, overflow, padding, text-align, text-shadow, transition